src/librbd/deep_copy/ObjectCopyRequest.cc

```diff
@@ -835,6 +861,35 @@ void ObjectCopyRequest<I>::compute_dst_object_may_exist() {
                    << dendl;
 }
 
+template <typename I>
+uint64_t ObjectCopyRequest<I>::compute_starting_end_size(){
+  uint64_t end_size = 0;
+
+  if (m_src_snap_id_start != 0) {
```

Comment:
nit: I would like to have something like `if (m_src_snap_id_start == 0) return 0` at the beginning to avoid additional nesting level (the code seems more readable to me then).

Reply:
Done.